src/OWML.ModHelper.Menus/ModMenu.cs

```diff
@@ -237,6 +237,11 @@ public object GetInputValue(string key)
 			{
 				return numberInput.Value;
 			}
+			var separator = GetSeparator(key);
+			if (separator != null)
+			{
+				return null;
```

Comment:
@salomj

Reply:
I thought about it, but separators don't really have a "value" associated with them. Would you like to return the whole separator objects or its title?

Reply:
¯\_(ツ)_/¯

Reply:
Yeah, I could write the code a bit cleaner to throw the warning if the input is not a separator. The reason the code is there is to prevent the warning on the next line from being displayed.

Reply:
Not sure if you like this better, but if you do, here is the change. https://github.com/amazingalek/owml/pull/405